.changeset/afraid-steaks-hang.md

```diff
@@ -0,0 +1,12 @@
+---
+"@marigold/components": patch
+"@marigold/system": patch
```

Comment:
Patch or BR? 😜 I think at least the themes should be a BR. For components, we only use the listbox internally so it is fine I guess